src/constants.py

```diff
@@ -11,12 +11,13 @@
 SPONSORBLOCK_MUSIC_API = 'https://sponsor.ajay.app/api/skipSegments?videoID={0}&category=music_offtopic'
 __config = configparser.ConfigParser()
 
+##### CONFIG F
+
 def generate_config():
     """Generates a default configeration file."""
     __config.add_section('Default')
     __config['Default']['TOKEN'] = ''
-    __config['Default']['FFMPEG_PATH'] = ''
-
+    __config['Default']['FFMPEG_PATH'] = 'ffmpeg/ffmpeg'
     __config.add_section('Cache')
```

Comment:
generate_config() does not need a change since the ffmpeg_path is dependent on platform.

The code below it needs to be better at finding the ffmpeg executable in the program root folder.

Reply:
Fixed code in Exsects/mbox.